python/subunit/tests/test_test_protocol2.py

```diff
@@ -17,6 +17,10 @@
 from io import BytesIO
 import datetime
 
+from hypothesis import given
+# from hypothesis import Settings, Verbosity
+# Settings.default.verbosity = Verbosity.verbose
```

Comment:
Mildly. Looking up docs is easy and reading code is relatively hard, so I'm all for killing the clutter. If you decide to keep the comment, I suggest adding `# To get more verbose output:` or similar (assuming that is what this code does!)